src/parseTree/nodes/node.java

```diff
@@ -0,0 +1,15 @@
+package src.parseTree.nodes;
+
+public interface node {
+    /**
+     * Adds a child to the node's internal list of children
+     *
+     * @param child The child to add to the node
+     */
+    public void addChild(Object child);
+
+    /**
+     * Assign children to their fields for execution
```

Comment:
kind of unclear on what this is supposed to do; maybe there's a better verb than fix?

Reply:
From my understanding, each node has a list of Objects. Based on the index, the Object is casted to a specific token/node and set to their respective fields. Maybe map or assign?